### DISCUSSION TITLE
gitlab.exceptions.GitlabListError: 403: 403 Forbidden

### DISCUSSION THREAD

**Discussion by lmilbaum** [python-gitlab/python-gitlab#2304] [General]
Invoking gl.runners.all() returns 403 while gl.runners.list() returns a list of runners. 
Does all() function require different permissions? If yes, maybe a documentation clarification is required.

**Comment by nejch** [comment]
Yes, listing all runners is restricted to instance administrators only, as described in the upstream docs for this endpoint:
https://docs.gitlab.com/ee/api/runners.html#list-all-runners

If you think it'd make sense to add this in the wrapper docs as well, I'd be happy to review PRs! :)

**Reply by lmilbaum** [reply]
Thanks @nejch. Will consider submitting one.
